lib/cards/basic_card.rb

```diff
@@ -0,0 +1,58 @@
+class BasicCard
+  include BankErrors
+
+  attr_accessor :balance
+  attr_reader :number
+  CARD_NUMBER_LENGTH = 16
+
+  TAX_PERCENT = {
+    withdraw: 0,
+    put: 0,
+    sender: 0
+  }.freeze
+
+  TAX_FIXED = {
+    withdraw: 0,
+    put: 0,
+    sender: 0
+  }.freeze
+
+  def balance!(amount)
+    @balance = amount
+  end
```

Comment:
imho u dont need this function u also have attr_accessor and can set value by card.balance = amount

Reply:
Thanks for pointing it out, but I'd rather get rid of attar_accessor as it's not safe and give you the option to put an invalid  input into @balance variable. Like ```balance = 'foobar'```. I think that it is a good idea to make a separate method for balance changing. Anyways, I've removed attr_accessor :balance.